server.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+const dotenv = require('dotenv').config();
+const helmet = require('helmet');
+const morgan = require('morgan');
+const db = require('./data/dbConfig.js');
+const session = require('express-session'); // brints in session library
+const knexSessionStore = require('connect-session-knex')(session);
+
+const sessionConfig = {
```

Comment:
good job keeping everything hidden and using .env for the info. 

also FANTASTIC job with all your commenting. Very impressive